packages/web-components/@sfx/autocomplete/test/autocomplete.test.ts

```diff
@@ -0,0 +1,72 @@
+import { expect, spy, stub } from './utils';
```

Comment:
Do we expect to have tests other than unit tests in this folder? If so, I would put this file in a `unit` subdirectory.